scripts/ops/add_meta_module.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ -z "$1" ]; then
+    echo "Error: No directory specified."
+    echo "Usage: $0 <data_directory>"
+    exit 1
+fi
+DATA_DIR=$1
+
+echo "This script will add the meta module to federation config, which is technically"
+echo "a consensus-breaking change. Make sure that:"
+echo "  * All fedimintd instances are shut down"
```

Comment:
Do they need to be gracefully shut down at the session closure?

Reply:
Technically yes, but afaik the module doesn't send CIs on its own, right? So as long as you don't start voting on stuff right away and let a session pass first we should be good. @joschisan what do you think (except for "NOOOOOOOO!" :laughing: )?